taskpools/chase_lev_deques.nim

```diff
@@ -111,13 +109,21 @@ proc grow[T](deque: var ChaseLevDeque[T], buf: var ptr Buf[T], top, bottom: int)
   for i in top ..< bottom:
     tmp[][i] = buf[][i]
 
-  buf.prev = deque.garbage
+  buf.prevRetired = deque.garbage
   deque.garbage = buf
   # publish globally
   deque.buf.store(tmp, moRelaxed)
   # publish locally
   swap(buf, tmp)
 
+proc garbageCollect(deque: var ChaseLevDeque) {.inline.} =
+  var node = deque.garbage
+  while node != nil:
+    let tmp = node.prevRetired
+    c_free(node)
+    node = tmp
+  deque.garbage = nil
```

Comment:
Does `garbage` need to be `Atomic` as well?

(Doesn't seem like it's necessary to me, but still worth to double-check intent)